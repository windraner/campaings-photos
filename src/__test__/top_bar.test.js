import React from 'react';
import TopBar from '../components/main_container/top_bar/top_bar';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

const mock = {
  isSelectedAll: false,
  isFilterShown: true,
  itemsView: 'list',
  switchHandler() {},
  stateHandler() {},
};

describe('TopBar', () => {
  it('matches the snapshot', () => {
    const tree = shallow(<TopBar {...mock} />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
