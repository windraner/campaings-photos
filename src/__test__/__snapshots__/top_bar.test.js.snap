// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar matches the snapshot 1`] = `
<div
  className="d-flex justify-content-between px-4 pb-3"
>
  <CustomTopBarCheckbox
    isChecked={false}
    switchHandler={[Function]}
  />
  <div
    className="d-flex justify-content-between"
  >
    <CustomTopBarSwipeCheckbox
      isChecked={true}
      switchHandler={[Function]}
    />
    <ViewSwitcher
      itemsView="list"
      stateHandler={[Function]}
    />
  </div>
</div>
`;
